src/main/webapp/WEB-INF/jsp/login.jsp

```diff
@@ -0,0 +1,28 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <title>Login | Personal Dashboard</title>
+        <link rel="stylesheet" type="text/css" href="${pageContext.request.contextPath}/css/style.css" />
+    </head>
+    <body>
+        <nav id="main-menu">
+        </nav>
+        <section>
```

Comment:
HTML5 introduced a couple of semantic tags to provide a better way to structure your HTML markup.
`<section>` should be used to mark a piece of your page **inside** the main content area.
Since the login form is your main content area here, a more semantic way would be to use the `<main>` tag instead.